tests/checkpointing/test_model_checkpoint.py

```diff
@@ -821,9 +800,7 @@ def test_model_checkpoint_save_last_checkpoint_contents(tmpdir):
     ckpt_last_epoch = torch.load(path_last_epoch)
     ckpt_last = torch.load(path_last)
 
-    # `-1` because this checkpoint is saved `on_train_epoch_end` which is considered part of the epoch so the
-    # `current_epoch` count has not been increased yet
```

Comment:
Both the "last" checkpoint and "top-k" checkpoint are now saved together.
We had a bug where we saved a duplicate "last" checkpoint `on_train_end`